lib/Net/Jabber/Bot.pm

```diff
@@ -1016,9 +1016,9 @@ sub _send_individual_message {
         return "Server is down.\n";
     }
 
-    # Strip out anything that's not a printable character
+    # Strip out anything that's not a printable character except new line, we want to be able to send multiline message, aren't we?
     # Now with unicode support?
-    $message_chunk =~ s/[^[:print:]]+/./xmsg; 
+    $message_chunk =~ s/[^[\n|\r|\r\n|[:print:]]]+/./xmsg;
```

Comment:
This is a character set. I need to check to be sure but I think you just want: `s/[^\r\n[:print:]]+/./xmsg; `

Reply:
You're right. I miss that. Thanks.